remote_challenge_evaluation/main.py

```diff
@@ -0,0 +1,108 @@
+import os
+import time
+
+import requests
+from eval_ai_interface import EvalAI_Interface
+from evaluate import evaluate
+
+auth_token = os.environ["AUTH_TOKEN"]  # Go to EvalAI UI to fetch your auth token
+evalai_api_server = os.environ[
+    "API_SERVER"
+]  # For staging server, use -- https://staging.eval.ai; For production server, use -- https://eval.ai
+queue_name = os.environ["QUEUE_NAME"]  # Check Manage Tab of challenge for queue name
+challenge_pk = os.environ[
+    "CHALLENGE_PK"
+]  # Check Manage Tab of challenge for challenge PK
+save_dir = os.environ.get("SAVE_DIR", "./")  # Location where submissions are downloaded
```

Comment:
Can you fix the code style here? The comments can be a line above the variable it doesn't need to be on the same line